### ISSUE TITLE
Decouple the establishment and reproduction/dispersal terms

### ISSUE THREAD

**Issue by agimenezromero** [ncsu-landscape-dynamics/rpops#189]
It would be interesting to allow that temperature and precipitation coefficients influenced independently the establishment and reproduction terms.

**Comment by ChrisJones687** [comment]
I agree that this could potentially be useful for some species. This could be added and would likely increase the computational memory that the model requires. Memory is the typical bottleneck for most people using the models workflow.
